protocol/shannon/fullnode_cache.go

```diff
@@ -136,11 +160,43 @@ func NewCachingFullNode(
 		evictionPercentage,
 	)
 
+	// Shared params cache
+	sharedParamsMinRefreshDelay, sharedParamsMaxRefreshDelay := getCacheDelays(sharedParamsCacheTTL)
+	sharedParamsCache := sturdyc.New[*sharedtypes.Params](
+		sharedParamsCacheCapacity,
+		1,
+		sharedParamsCacheTTL,
+		evictionPercentage,
+		sturdyc.WithEarlyRefreshes(
+			sharedParamsMinRefreshDelay,
+			sharedParamsMaxRefreshDelay,
+			sharedParamsCacheTTL,
+			retryBaseDelay,
+		),
+	)
+
+	// Block height cache
+	blockHeightMinRefreshDelay, blockHeightMaxRefreshDelay := getCacheDelays(blockHeightCacheTTL)
```

Comment:
Given the consistent interval on which block height changes we may want to consider making the blockHeight delay a const rather than calculating based on the same percentages as the other caches, ie. make it early refresh at a consistent 15 seconds. Given we only cache one entry for block height I don't think we need to worry about thundering herd issues, which is main reason for staggering the early cache refresh for the session cache.